docs/update.md

```diff
@@ -20,27 +20,38 @@ Distribution for PostgreSQL *without* the Operator upgrade.
 
 ## Upgrading the Operator and CRD
 
-The Operator supports only the incremental update to its nearest version (such
-as updating the Operator from 2.4.0 to 2.4.1).
-To update to a newer version, which differs from the current version by more
-than one, make several incremental updates sequentially. For example, to upgrade
-the Operator and CRD from the version 2.3.0 to 2.4.1, you need to do the
-following sequence of upgrades:
+The Operator version includes three numbers: `major`, `minor`, and `patch` (for
+example, the Operator version `2.6.0` has major version `2`, 
+minor version `6`, and patch version `0`). Only the incremental update to a
+nearest `major.minor` version of the Operator is supported. To update to a newer
+version, which differs from the current `major.minor` version by more than one,
+make several incremental updates sequentially.
 
-1. upgrading the Operator and CRD from 2.3.0 to 2.3.1,
-2. upgrading from 2.3.1 to 2.4.0,
-3. upgrading from 2.4.0 to 2.4.1.
+For example, to upgrade the CRD and Operator from the version 2.4.0 to 2.6.0,
+the following sequence of upgrades will be the shortest recommended path:
 
-You can find Operator versions [listed here](ReleaseNotes/index.md).
+1. upgrading the CRD and Operator from 2.4.0 to 2.5.1,
+2. upgrading from 2.5.1 to 2.6.0.
 
-!!! note
+You can find Operator versions [listed here](ReleaseNotes/index.md).
 
-    The Operator supports **last 3 versions of the CRD**, so it is technically
-    possible to skip upgrading the CRD and just upgrade the Operator. If the CRD
-    is older than the new Operator version *by no more than three releases*, you
-    should be able to continue using the old CRD and even carry on Percona Distribution
-    for PostgreSQL minor version upgrades with it. But updating the Operator *and* CRD
-    is the **recommended path**.
+The Operator supports **last 3 versions of the CRD**, so it is technically
+possible to skip upgrading the CRD and just upgrade the Operator. If the CRD
+is older than the new Operator version *by no more than three releases*, you
+should be able to continue using the old CRD and even carry on Percona Distribution
+for PostgreSQL patch version upgrades with it. But updating the CRD *and* Operator 
+is the **recommended path**. 
```

Comment:
i am not sure about this. if we add a new field (which we frequently do) it won't work without CRD upgrade. I think CRD should be upgraded with every release but newest CRD should work with last three versions of operator. @spron-in @hors wdyt?

Reply:
@egegunes I agree, the idea is that the latest CRDs should work with last three versions of operator

Reply:
I agree.

Reply:
Ok, I'll change this one, and then we spread the change to all other operators